lib/devise/token_authenticatable/model.rb

```diff
@@ -36,7 +36,13 @@ module Models
     #
     module TokenAuthenticatable
       extend ActiveSupport::Concern
-
+      
+      Devise::Models.config(self, :should_reset_authentication_token, :should_ensure_authentication_token)
```

Comment:
At https://github.com/baschtl/devise-token_authenticatable/pull/13/files#diff-b8db9b4dfd853c279e1f6a0c5802cbbfR92 there is already a configuration for model attributes. Please move these to down there.